app/presenters/order_detail_presenter.rb

```diff
@@ -21,6 +21,10 @@ def description_as_html
     end.join(" &mdash; ").html_safe
   end
 
+  def description_as_html_with_facility_prefix
+    "#{facility.abbreviation} / #{description_as_html}".html_safe
```

Comment:
Does this really need to be `html_safe`d? Or is it something that goes through the markdown smart-quote filter that we could turn off?